package.json

```diff
@@ -4,23 +4,23 @@
   "author": "Maxime Robert <maxime.robert1992@gmail.com>",
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
-    "body-parser": "1.12.3",
-    "compression": "1.4.3",
-    "cors": "2.7.1",
-    "dompurify": "0.8.1",
-    "express": "4.13.4",
-    "express-validator": "2.20.5",
-    "jsdom": "9.2.1",
-    "jsonwebtoken": "7.0.0",
-    "mongoose": "4.4.19",
-    "morgan": "1.5.2",
+    "body-parser": "^1.16.0",
+    "compression": "^1.6.2",
+    "cors": "^2.8.1",
+    "dompurify": "^0.8.4",
+    "express": "^4.14.0",
+    "express-validator": "^3.1.2",
+    "jsdom": "^9.9.1",
+    "jsonwebtoken": "^7.2.1",
+    "mongoose": "^4.7.7",
+    "morgan": "^1.7.0",
     "passport": "0.3.2",
     "passport-http-bearer": "1.0.1",
-    "socket.io": "1.4.6",
-    "spdy": "3.4.0"
+    "socket.io": "^1.7.2",
+    "spdy": "^3.4.4"
   },
   "devDependencies": {
-    "babel-core": "6.9.1",
-    "babel-preset-es2015": "6.9.0"
+    "babel-core": "^6.21.0",
+    "babel-preset-es2015": "^6.18.0"
```

Comment:
@JerimSD could you please remove all the `^` ?
(and run `yarn` right after to update the `yarn.lock` file)

Thanks